<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IStatusDEM</class>
 <widget class="QWidget" name="IStatusDEM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>25</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QRadioButton" name="radioNone">
     <property name="text">
      <string>none</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioShading">
     <property name="text">
      <string>shading</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioContour">
     <property name="text">
      <string>contour</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioSlope">
     <property name="text">
      <string>slope</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
